configuration/scripts/tests/base_suite.ts

```diff
@@ -9,7 +9,7 @@ smoke          gx3     7x2        diag1,bigdiag,run1day,diagpt1
 decomp         gx3     4x2x25x29x5  none
 smoke          gx3     4x2        diag1,run5day             smoke_gx3_8x2_diag1_run5day
 smoke          gx3     4x1        diag1,run5day,thread      smoke_gx3_8x2_diag1_run5day
-smoke          gx3     1x8        diag1,run5day,evp1d       smoke_gx3_8x2_diag1_run5day
+smoke          gx3     1x8        diag1,run5day,evp1d
```

Comment:
what is the justification for removing the b4b compare with `smoke_gx3_8x2_diag1_run5day` ?

Reply:
Because it fails and is expected to fail.  We would have to turn on the debug flag for the baseline and test to have a chance of bit-for-bit and that would not happen for all compilers.  After watching it fail a few times, I decided to just remove it.  We could/should consider adding a debug version of the test though.